modules/auth/supabase/confirm-reset-password.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { useTranslations } from "next-intl";
+import ConfirmResetPasswordForm from "@/core/forms/confirm-reset-password-form";
+import { getStaticPropsWithTranslations } from "@/modules/lang/props";
+import { GetStaticProps } from "next";
+import { GlobalCore } from "@/core/module/module.types";
+import auth_style from "./styles/auth.module.css";
+
+export const getStaticProps: GetStaticProps = getStaticPropsWithTranslations;
+
+const ConfirmResetPassword = () => {
```

Comment:
Can not we use the same component `reset-password` with a propthat define is reset or confirm? What was decision behind a new component?

Reply:
It was a bit messy especially because this page in redirected to from supabase backend not from the frontend